test/DXILValidation/Metadata/unknown_nodeio_tags.ll

```diff
@@ -0,0 +1,84 @@
+; RUN: not %dxv %s | FileCheck %s
```

Comment:
I'd like to ask why the `not` command is necessary here? Why not just `%dxv`?

Reply:
This is expected to fail validation. That makes the result of the dxv invocation produce a failing errorcode. If that happens, the test fails outright and won't pipe the results into filecheck at all.

Reply:
There are other examples of validation failing and returning bad exit codes, but the output still gets piped to filecheck. Is this required just because of how lit works?

Reply:
Yes. taef FileCheck will take care the failure.
But lit FileCheck test will need to use 'not'.